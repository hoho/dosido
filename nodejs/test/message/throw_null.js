'use strict';
var common = require('../common');
var assert = require('assert');

throw null;
