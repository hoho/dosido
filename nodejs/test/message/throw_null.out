
*test*message*throw_null.js:5
throw null;
^
null
